docs/events.md

```diff
@@ -1,50 +1,122 @@
-# Events
-
-## XACC Tech Talks
-
-The XACC Tech Talks are a series of webinars covering a broad range of topics related to Xilinx Adaptive Compute. Each session will be 1 hour, with two 30-minute talks in each session. 
 
-### Upcoming Talks
 
-| Tech Talk 3: 8 July 2021 15:00-16:00 (CET)                   |
-| ------------------------------------------------------------ |
-| Blockchain Machine: Accelerating Validation Bottlenecks in Hyperledger Fabric; Dr. Haris Javaid, Xilinx |
-| ThunderGP: HLS-based Graph Processing on FPGAs ; Xinyu Chen, National  University of Singapore |
+# Events
 
-[Details and registration](./xacc_tech_talks.html#xacc-tech-talk-3)
+### XACC Tech Talks
 
-| Tech Talk 4: 22 July 2021 17:00-18:00 (CET)                  |
-| ------------------------------------------------------------ |
-| Fletcher: A framework for high-performance big data analytics using FPGAs;  Joost Hoozemans, TU Delft |
-| ThundeRiNG: Generating Multiple Independent Random Number Sequences on FPGAs; Hongshi Tan, National  University of Singapore |
+The XACC Tech Talks are a series of webinars covering a broad range of topics related to Xilinx Adaptive Compute. Each session will be 1 hour, with two 30-minute talks in each session. 
 
-[Details and registration](./xacc_tech_talks.html#xacc-tech-talk-4)
 
-### Previous Talks (watch the recording)
 
-| XACC Tech Talk 1: 10 June 2021 | Recording                                           |
-| ------------------------------------------------------------ | --------------------------------------------------- |
-| Coyote: Do OS abstractions make sense in FPGAs? Dario Korolija, ETH Zurich | [https://www.youtube.com/watch?v=un7wck0IkGs&t=88s](https://www.youtube.com/watch?v=un7wck0IkGs&t=88s)   |
-| Data-Centric FPGA Programming with Multi-Level Design; Johannes de Fine Licht, ETH Zurich | [https://www.youtube.com/watch?v=un7wck0IkGs&t=1743s](https://www.youtube.com/watch?v=un7wck0IkGs&t=1743s) |
+## Upcoming Talks
+
+<table>
+    <tr>
+        <th style="text-align:center; background:transparent"; colspan="2">
+            <font size="+2">
+                <strong>Tech Talk 4: 22<sup>nd</sup> July 2021 17:00-18;00(CET)</strong>
+            </font>
+        </th>
+    </tr>
+    <tr>
+        <td style="background:transparent">
+            <strong>Fletcher:</strong> A framework for high-performance big data analytics using FPGAs;  Joost Hoozemans, TU Delft  
+        </td>
+        <td style="text-align:center; background transparent"; rowspan="2"; width="200">
+            <a href="(./xacc_tech_talks.html#xacc-tech-talk-4)">Details and Registration</a>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <strong>ThundeRiNG:</strong> Generating Multiple Independent Random Number Sequences on FPGAs; Hongshi Tan, National  University of Singapore
+        </td>
+    </tr>
+
+
```

Comment:
table is not closed here, this seems to cause this table to be under previous talks